content/docs/0.11.x/operate/k8s_support/index.md

```diff
@@ -21,14 +21,14 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 
 | Keptn Version /<br>Installation                           | Kubernetes  | AKS                       | EKS                       | GKE           | OpenShift   | K3s         | Minishift               |
 |-----------------------------------------------------------|:-----------:|:-------------------------:|:-------------------------:|:-------------:|:-----------:|:-----------:|:------------------------|
-| **0.11.x** / <br>Control & Execution plane<br>*see: (3)*   | 1.21 - 1.16 | 1.20 - 1.15<br>*see: (1)* | 1.18 - 1.16<br>*see: (2)* | 1.19 - 1.15   | 4, 3.11     | 1.21 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
-| **0.11.x** / <br>Control plane                             | 1.21 - 1.16 | 1.20 - 1.15<br>*see: (1)* | 1.18 - 1.16<br>*see: (2)* | 1.19 - 1.15   | 4, 3.11     | 1.21 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
+| **0.11.x** / <br>Control & Execution plane<br>*see: (2)*   | 1.21 - 1.19 | 1.21 - 1.19 | 1.18 - 1.16<br>*see: (1)* | 1.21 - 1.19   | 4, 3.11     | 1.21 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
+| **0.11.x** / <br>Control plane                             | 1.21 - 1.19 | 1.21 - 1.19 | 1.18 - 1.16<br>*see: (1)* | 1.21 - 1.19   | 4, 3.11     | 1.21 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
 
 **Remarks:**
 
-* (1): AKS with K8s version before 1.15 might fail (see: [#1429](https://github.com/keptn/keptn/issues/1429)), due to a known AKS issue: [#69262](https://github.com/kubernetes/kubernetes/issues/69262)
-* (2): EKS did not provide K8s 1.19 clusters for testing (checked on: 3rd March, 2021)
-* (3): Requires sufficient resources (e.g., >= 8 vCPUs and 14 GB memory for deploying sockshop in multiple stages) depending on your use-case and workloads.
+
+* (1): EKS did not provide K8s 1.19 clusters for testing (checked on: 3rd March, 2021)
```

Comment:
EKS is on 1.19 now (https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html). Can we remove this line please.

Reply:
EKS is not tested yet - we stay with the K8s we have right now.